src/components/modal/v2/styles/components/_donut.scss

```
@@ -93,6 +101,57 @@ $grey600: #888;
             @include desktop {
                 line-height: 22px;
             }
+            white-space: nowrap;
+            &_v4 {
+                @extend .donut__timestamp;
+                color: $donut-text-v4;
+            }
+        }
+    }
+    &_line {
+        &:first-child {
+            svg g {
+                fill: $donut-background;
+            }
+            span.donut__timestamp_v4,
+            span.donut__payment_v4 {
+                color: $text-main;
+            }
+        }
+
+        &::after {
+            content: '';
+            flex-grow: 1;
+            border-top: 1px dashed $grey350;
+            top: 55px;
+            width: 100%;
+            max-width: 50px;
+            position: absolute;
+            transform: translateX((90%));
+
+            @include mobile {
+                max-width: 45px;
+            }
+            @include smallMobile {
+                max-width: 35px;
+            }
+        }
+        &__3 {
+            @extend .donut__single_payment_line;
+            &::after {
+                @include tablet {
+                    max-width: 85px;
+                    transform: translateX((70%));
+                }
+                @include mobile {
+                    max-width: 70px;
+                    transform: translateX((75%));
+                }
```

```suggestion
                @include mobile {
                    max-width: 87px;
                    transform: translateX((54px));
                }
```
This seems to play a little bit nicer with mobile devices for Pi3, rather than trying to perfectly center the line and spacing between donuts which is definitely difficult with the different sizes.